parser-typechecker/src/Unison/Codebase/SqliteCodebase/Operations.hs

```diff
@@ -524,3 +534,62 @@ declExists = termExists
 before :: Branch.Hash -> Branch.Hash -> Transaction (Maybe Bool)
 before h1 h2 =
   Ops.before (Cv.causalHash1to2 h1) (Cv.causalHash1to2 h2)
+
+namesWithinPath ::
```

Comment:
I think this may fall just below my (arbitrary) threshold for, "does the name + type say enough that it doesn't need a comment?".

I guess one question I have is, what does `Nothing` here mean? -- My guess is it doesn't mean the root path, since that can be represented as `[]`.

Reply:
Yeah I think originally I'd planned for it to affect how names were made to be relative in the root-path case, but I'm pretty sure it's not necessary anymore. I'll remove the Maybe.